test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const fetch = require('node-fetch');
+
+describe('Login Page', () => {
+    it('should return 200 for valid credentials', async () => {
+        const res = await fetch('http://localhost:3000/api/login', {
+            method: 'POST',
```

Comment:
The test cases are using hardcoded values for the API endpoint and the test credentials. Consider using variables or constants to store these values to make the tests more flexible.